### ISSUE TITLE
Using "Tab" key when creating a new account doesn't activate the submit button

### ISSUE THREAD

**Issue by ogrotten** [joincomet/comet#87]
How to reproduce:

1. Create a new account
2. Type name, press tab.
3. Type email, press tab.
4. Type password, press tab.
5. Type confirm password, press tab.
6. Submit button doesn't activate.

How to avoid:

1. Create new account
2. Click name field, type name.
3. Click all the other fields, individually, in order and fill them out, in order. 
4. Submit button activates.

Expected behaviour:

* Tab key should not have a different effect than mousing.

**Comment by insuusvenerati** [comment]
I am unable to reproduce this. What browser are you using? Can you try without any extensions?

**Comment by ogrotten** [comment]
Hm... it's weird. I tried it a couple times just now and discovered that it's pretty specific. Here's a screen cap.

![0614 joincomet](https://user-images.githubusercontent.com/8119012/121929211-921ffd80-cd06-11eb-9d43-02f6aa96eab0.gif)

TLDR: if you go to a url, then modify the url in the bar, that's when it misbehaves. following a link or even pasting `joincomet.app` does not reproduce it.

**Comment by ogrotten** [comment]
What the . . . I'm not even sure anymore. I just tried it again and it worked. Then I tried it again using the same user/email/pass as I used in the screencap and it didn't work.

user: hambone
email: hambone@example.com
pass: mynameis

I looked at the code > routes, thinking maybe there was some kinda route fall-thru that wasn't catching it, but with my limited knowledge of the codebase it appears to be AOK. `/` and noslash work the same with the `hambone` new account, but others (I don't remember what they were, just random thoughts) did not have a problem.
